tools/autograd/gen_variable_type.py

```diff
@@ -934,10 +934,16 @@ def load_aten_declarations(path):
         declaration['base_name'] = declaration['name']
 
         # if the return value is missing a name, call it 'output'
```

Comment:
Shouldn't this go in `gen.py`? `Declarations.yaml` is intended to be pre-sanitized, and making sure everything always has an output name seems like a sanitation pass.

Reply:
maybe in preprocess_declarations make more sense?

Reply:
actually, this is hard to do before -- I'll just put it in a postprocess step in gen.py.